package.json

```diff
@@ -17,7 +21,16 @@
   },
   "homepage": "https://github.com/FACK1/fid-sh-had-week8#readme",
   "dependencies": {
+    "compression": "^1.7.3",
     "env2": "^2.2.2",
+    "eslint": "^5.10.0",
```

Comment:
```eslint``` should be inside devDependencies. It's not required to run it in production mode